teuthology/suite/run.py

```diff
@@ -29,6 +29,7 @@
 class Run(object):
     WAIT_MAX_JOB_TIME = 30 * 60
     WAIT_PAUSE = 5 * 60
+    MAX_JOBS = 500
```

Comment:
Nit: `MAX_JOBS` is a bit misleading IMO -- it's not a hard limit but may be interpreted as such.  I would just hard-code `500` in `check_num_jobs()`, similar to `check_priority()`.

Reply:
maybe DEFAULT_JOB_LIMIT is better for this "const"?

Reply:
I think DEFAULT_JOB_LIMIT makes it more clear, updated the commit.